src/components/multi_select.js

```diff
@@ -104,7 +104,11 @@ export class MultiSelect extends PureComponent {
       searchValue,
       withGrouping,
       generateClassName,
-      listRenderer
+      listRenderer,
+      showSelectedItemsSearch,
+      searchSelectedItemsValue,
+      filterSelectedItems,
```

Comment:
I don't think we need a separate filter function for destination list. I think it will be enough to use filterItems for both source and destination lists